client/message/history.jsx

```diff
@@ -31,6 +31,17 @@ const History = React.createClass({
             myUsername: null
         };
     },
+    deleteLiveMessage(username) {
```

Comment:
We don't use the term "live" anywhere else. Maybe we can name this "deleteTypingMessage".